webapp/pages/index.vue

```diff
@@ -36,23 +36,24 @@ import FilterMenu from '~/components/FilterMenu/FilterMenu.vue'
 import gql from 'graphql-tag'
 import uniqBy from 'lodash/uniqBy'
 import HcPostCard from '~/components/PostCard'
-import HcLoadMore from '~/components/LoadMore.vue'
+import InfiniteLoading from 'vue-infinite-loading'
 import { mapGetters } from 'vuex'
 
 export default {
   components: {
     FilterMenu,
     HcPostCard,
-    HcLoadMore,
+    InfiniteLoading,
   },
   data() {
     const { hashtag = null } = this.$route.query
     return {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-03T16:21:48Z" title="Wednesday, July 3rd 2019, 6:21:48 pm +02:00">Jul 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -60,7 +61,18 @@ export default {
       return (this.page - 1) * this.pageSize
     },
   },
+  mounted() {
+    this.checkScroll();
+  },
   methods: {
+    checkScroll() {
+      window.onscroll = () => {
```


Try to avoid globals like `window`